src/main/java/com/bloomtechlabs/fp/entities/EducationHistory.java

```diff
@@ -0,0 +1,88 @@
+package com.bloomtechlabs.fp.entities;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name ="education_histories")
+public class EducationHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "client_id")
+    private Long clientId;
+
+    @Column(name = "school_name")
+    private String schoolName;
+
+    @Column(name = "level")
+    private String level;
+
+    @Column(name = "start_date")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date startDate;
+
+    @Column(name = "end_date")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date endDate;
+
+    public EducationHistory() {
+    }
+
+    public EducationHistory(Long clientId, String schoolName, String level, Date startDate, Date endDate) {
```

Comment:
Do we want to include an input parameter for clientId if the table is auto generating the value?

Reply:
yes, that is the clientId that we got from the clients table in the schema. the id that was auto-generated in this entity was not included.

Reply:
Oh I see. That makes sense. ClientId is a foreign key from the client table. Thanks for clearing that up.

Reply:
Looking at the schema the clientId is also UUID so I may need to change that also